roles/setup/defaults/main.yml

```diff
@@ -1,19 +1,18 @@
 admin_url: http://app:8000/admin
 auth_client_id: admin-cli
-auth_keycloak_url: '{{ lookup(''env'', ''PINAKES_KEYCLOAK_URL'') | default(''http://default_keycloak:8080/auth'',
-  True) }}'
-auth_password: '{{ lookup(''env'', ''PINAKES_KEYCLOAK_PASSWORD'') | default(''admin'', True)
-  }}'
-auth_realm: '{{ lookup(''env'', ''PINAKES_KEYCLOAK_MASTER_REALM'') | default(''master'', True) }}'
-auth_username: '{{ lookup(''env'', ''PINAKES_KEYCLOAK_USER'') | default(''admin'', True) }}'
+auth_keycloak_url: "{{ lookup('env', 'PINAKES_KEYCLOAK_URL') | default('http://default_keycloak:8080/auth', True) }}"
```

Comment:
@Alex-Izquierdo Generally speaking is it advisable for role to use environment variables directly? Or should the role be configured by the playbook, with appropriate environment variables?

Reply:
I think @mkanoor did it in that way to take the values from the dev environments. As long as have default values I don't see any problem.